CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Full documentation for rocPRIM is available at [https://rocm.docs.amd.com/projects/rocPRIM/en/latest/](https://rocm.docs.amd.com/projects/rocPRIM/en/latest/).
 
+## (Unreleased) rocPRIM 3.4.x for ROCm 6.4.x
```

Comment:
Please change this to rocPRIM 3.4.0 for ROCm 6.4.0

Reply:
Changed this and the base branch